### ISSUE TITLE
Include Docker HealthCheck in Zwave-Js-UI Docker Image

### ISSUE THREAD

**Issue by barndawgie** [zwave-js/zwave-js-ui#3054]
**Is your feature request related to a problem? Please describe.**
I like to have a HealthCheck for all of my containers so I know if they go into a bad state for some reason.

**Describe the solution you'd like**
Include a HealthCheck in the Docker file, or even just a simple healthcheck command that could be invoked via Docker Run or Docker-Compose configuration.

**Describe alternatives you've considered**
I tried simply using curl to run one myself (`curl -f -k -H "Accept: text/plain" https://localhost/health || exit 1`) but curl doesn't appear to be included in the container.

**Additional context**

**Comment by robertsLando** [comment]
Hi @barndawgie,

We spoke about this with users a while ago, the problem is that the healthcheck would return false when the container is not initialized, also there are some users that are using mqtt and some that are just using zwave client, in the first case they would check both are working. This is why we decided to just implement the healthcheck endpoint and leave user free to implement his preferred healthcheck method

cc @chrisns

**Comment by robertsLando** [comment]
BTW @barndawgie if you have any suggestion about how we could improve this let me know 🙏🏼

**Comment by barndawgie** [comment]
Thanks for the quick feedback, guys! That wget command did work, though I needed to add a --no-check-certificate and the local port number:
`--health-cmd='wget --no-verbose --spider --no-check-certificate --header "Accept: text/plain" https://localhost:8091/health || exit 1`

I understand why you don't want to include it natively, though I was able to work around any initialization issue with a  `--health-start-period=30s`.  Might be worth adding the suggested command/considerations to the docsumentation.
